docs/source/standard.md

```diff
@@ -255,39 +255,41 @@ The ability of the organisation to support users who want to browse the data ava
+A TRE conforming to the SATRE standard should enact broadly similar measures to protect against the unauthorised use of information, especially electronic data. These measures include vulnerability management of TRE infrastructure (whether physical or virtual/cloud-based), carrying out compliance checks and security tests of the TRE, common approaches to data encryption, and (where appropriate) physical security measures to prevent unauthorised access to the TRE .
 
-### 4.1 Vulnerability management
+### 4.1 Vulnerability management and security testing
 
-The ability of the organisation to identify, assess, report on, manage and remediate cyber vulnerabilities across endpoints, workloads, and systems.
+Vulnerability management describes the ability of the organisation operating a TRE to identify, assess, report on, manage and remediate cyber vulnerabilities across endpoints, workloads, and systems. Security testing enables the organisation to gain assurance in the security of a TRE by testing or attempting to breach some or all of that system's security.
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+| Statement                                                                                                                                   | Guidance                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| ------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Regular vulnerability scans of TRE infrastructure should be conducted                                                                       | Ensuring that scans are done on a regular basis can enable TRE operators can identify and address weaknesses that may have been introduced during the TRE's operational lifetime. One way this can be achived is by ensuring that all machines (whether physical or virtual/cloud-based) are set to check for operating system updates on a regular basis, and install any security patches pushed by the developers of those operating systems. |
```

Comment:
Is keeping a system up-to-date with OS updates sufficient? I think probably not.